src/stirling/utils/linux_headers.cc

```diff
@@ -627,9 +623,18 @@ Status InstallPackagedLinuxHeaders(const std::filesystem::path& lib_modules_dir)
   PX_ASSIGN_OR_RETURN(PackagedLinuxHeadersSpec packaged_headers,
                       FindClosestPackagedLinuxHeaders(kPackagedHeadersRoot, kernel_version));
   LOG(INFO) << absl::Substitute("Using packaged header: $0", packaged_headers.path.string());
-  PX_RETURN_IF_ERROR(ExtractPackagedHeaders(&packaged_headers));
+  std::string version =
+      absl::Substitute("$0.$1.$2-pl", packaged_headers.version.version,
+                       packaged_headers.version.major_rev, packaged_headers.version.minor_rev);
+  std::string staging_directory = absl::Substitute("/usr/src/staging/linux-headers-" + version);
+  std::string expected_directory = absl::Substitute("/usr/src/linux-headers-" + version);
```

Comment:
It looks like use of `absl::Substitute` is not needed here. We could take this in a few directions:
1. Don't use `absl::Substitute`.
2. `absl::Substitute("/usr/src/linux-headers-$0", version);`
3. Use `absl::StrCat`.

Reply:
Switched to absl::StrCat!